study.md

```diff
@@ -40,5 +44,5 @@ In your own words explain what a prototype is.  In your answer, be
 sure to cite any relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
+Prototype is an attribute that is attached with a property and method when a user wants to implement inheritance , also it is an attribute that points to the object's parent.
```

Comment:
Prototype is an object that stores methods that will be shared with any object that inherits from it.